pkg/operator/vspherecontroller/driver_starter.go

```diff
@@ -113,7 +107,6 @@ func (c *VSphereController) createCSIDriver() {
 			c.apiClients.ConfigMapInformer.Informer(),
 			c.apiClients.NodeInformer.Informer(),
 		},
-		WithVSphereCredentials(defaultNamespace, cloudCredSecretName, c.infraLister, c.configMapLister, c.apiClients.SecretInformer),
```

Comment:
Credentials are not longer set as env variables in the container, so I don't need a function hook anymore.